### ISSUE TITLE
error when zip not present

### ISSUE THREAD

**Issue by ryanburnette** [webinstall/packages#38]
I was installing webi on WSL/Ubuntu in Windows. I got this error.

```
ryan@RYAN-WIN10:~$ webi pathman
Error: no 'pathman' release for 'windows' on 'amd64' as one of 'tar,exe,xz,git,dmg' by the tag ''
       'pathman' is available for 'linux,macos,windows' on 'amd64' as one of 'zip'
       (check that the package name and version are correct)

       Double check at https://webinstall.dev/api/releases/pathman@.tab
```

I installed zip.

```
sudo apt install zip
```

Then it installed. But it wasn't that clear about what was going wrong.

**Comment by solderjs** [comment]
This is about as specific as I can think to make it in a generic way:

```
Error: no 'pathman' release for 'windows' on 'amd64' as one of 'tar,exe,xz,git,dmg' by the tag ''
       'pathman' is available for 'linux,macos,windows' on 'amd64' as one of 'zip'
```

Perhaps this would be better?

```txt
Error: You are on a 'windows' OS on a 'amd64' CPU and can install packages of type(s) 'tar,exe,xz,git,dmg'
       'pathman' is available for 'linux,macos,windows' on 'amd64' as one of 'zip', which your system doesn't support
```

That said, Linux packages are _ALWAYS_ available as `tar.gz` or `tar.xz`, so zip shouldn't be needed.

**Comment by ryanburnette** [comment]
The error message was probably confusing to me because it is derived from the logic of the install script.

```
Error: no 'pathman' release for 'windows' on 'amd64' as one of 'tar,exe,xz,git,dmg' by the tag ''
       'pathman' is available for 'linux,macos,windows' on 'amd64' as one of 'zip'
```

What it means in plain English is that your windows amd64 environment supports tar, exe, yz, git, and dmg, but this package is zip, and you don't support that.

I don't think we need to change anything, but it is good that this issue exists to help someone else who runs across this issue.
